shared/utility/support/math_string.cpp

```diff
@@ -40,6 +40,11 @@ namespace support {
         table.add_constants();
         table.add_constant("auto", std::numeric_limits<double>::infinity());
 
+        // File size constants
+        table.add_constant("KiB", 1024);
+        table.add_constant("MiB", 1048576);
+        table.add_constant("GiB", 1073741824);
```

Comment:
Is it worth replacing the raw values with `std::pow(2, 10)`, etc? Or maybe as a [hex floating point literal](https://en.cppreference.com/w/cpp/language/floating_literal)?
```
0x1p10 = 1024
0x1p20 = 1048576
0x1p30 = 1073741824
```

Reply:
Maybe, but these are pretty safe